src/data/staticData/extensionData.tsx

```diff
@@ -405,11 +406,24 @@ const extensions: { [key: string]: ExtensionData } = {
     currentVersion: 1,
     createdAt: 1603915271852,
     neededColonyPermissions: [ColonyRole.Root],
-    tokenContractAddress: (
+    tokenContractAddress: (tokenAddress) => (
       <FormattedMessage
         {...MSG.coinMachineTokenContractAddress}
         values={{
-          link: <ExternalLink href={BLOCKSCOUT_LINK} text="Blockscout" />,
+          link: (
+            <ExternalLink
+              href={getBlockExplorerLink({
+                linkType: 'token',
+                addressOrHash: tokenAddress,
+              })}
+              text={
+                DEFAULT_NETWORK === Network.Xdai ||
+                DEFAULT_NETWORK === Network.XdaiFork
+                  ? 'Blockscout'
+                  : 'Etherscan'
+              }
```

Comment:
Or localhost if `DEFAULT_NETWORK === Network.Local`

But that's just mostly for us...